src/gamepad/gamepad-mappings-manager.vala

```diff
@@ -34,6 +41,12 @@ private class Games.GamepadMappingsManager : Object {
 		add_from_input_stream (resources_open_stream (path, ResourceLookupFlags.NONE));
 	}
 
+	public void add_from_file (string file_name) throws Error {
+		var file = File.new_for_path (file_name);
+		if (!file.query_exists ()) return;
```

Comment:
Then a warning will be printed in the console every time because it is not necessary to have the file. Can i do something in the catch above to detect not a file?